object-cache.php

```diff
@@ -732,15 +740,20 @@ public function set( $key, $data, $group = 'default', $expire = 0 ) {
 	 * key and the data.
 	 */
 	public function stats() {
-		echo '<p>';
-		echo '<strong>Cache Hits:</strong>' . (int) $this->cache_hits . '<br />';
-		echo '<strong>Cache Misses:</strong>' . (int) $this->cache_misses . '<br />';
-		echo '</p>';
-		echo '<ul>';
+		$out = array();
+		$out[] = '<p>';
+		$out[] = '<strong>Cache Hits:</strong>' . (int) $this->cache_hits . '<br />';
```

Comment:
`$out[] = sprintf( '<strong>Cache Hits:</strong>%d<br />', $this->cache_hits );`